### ISSUE TITLE
[BUG] Shiny luck

### ISSUE THREAD

**Issue by EmberCM** [pagefaultgames/pokerogue#1586]
**Describe the bug**
TL;DR A fused shiny bugged my shop luck (maybe only visual)
My luck was 13 from 3 fused Pokemon (1/2 + 2/3 + 3/2). I then found a wild Epic shiny fused Gardevoir/Gyrados which I caught. When looking at the Gardevoir/Gyrados info page, the luck amount wasn't shown at all. It didn't say "Luck: 3" like the other Pokemon. When looking at the shop, the luck indicator would be blank showing "Luck: " with nothing after it. After unsplicing, the shop displayed "Luck: SSS" and the Pokemon info page shows "Luck: 3"

**To Reproduce**
1. Have 13 shiny luck
2. Catch a fused Epic Shiny that secondary Pokemon is not shiny and add it to the party, not replacing another shiny

**Expected behavior**
1. Shop Luck should show "SSS"
2. Fused Epic Shiny Gardevoir/Gyrados info page should show "Luck: 3"

**Screenshots / Videos**
I forgot to take more screenshots unfortunately
Team after unsplicing Gardevoir: ![chrome_pa7sT1hLGZ](https://github.com/pagefaultgames/pokerogue/assets/17890775/c847713e-b085-44dc-a01f-e5d1ba6573e8)

**Device**
Web Browser - Google Chrome

**Additional context**
N/A

**Comment by Smewkie** [comment]
As an avid player of Spliced Endless, I can confirm this one, seems to be any caught shiny fusion that is added to the team will have a bugged luck value, refreshing the page or unfusing the caught pokemon with the bugged luck value will fix this, but it is a bug worth looking in to.

**Comment by EmberCM** [comment]
The bug occurs when catching any fusion Pokemon, it doesn't need to be shiny. I caught a non-shiny fusion Pokemon and my shop luck glitched again.
![luck](https://i.ember.zone/xk3DhgBBKiTaHmTZ.png)
![team](https://i.ember.zone/0RkddJYkzCjZb3Bs.png)

**Comment by Tempo-anon** [comment]
This should be fixed now
